crates/shared/src/price_estimation/native_price_cache.rs

```diff
@@ -231,8 +237,10 @@ async fn update_recently_used_outdated_prices(
     update_interval: Duration,
     update_size: Option<usize>,
     max_age: Duration,
+    concurrent_requests: usize,
 ) {
     while let Some(inner) = inner.upgrade() {
+        tracing::error!("run background task");
```

Comment:
Left-over log used for debugging?

Reply:
Yeah, this one slipped through the cracks.
Removed in [7351ee7](https://github.com/cowprotocol/services/pull/1054/commits/7351ee736d8209aa9ae17a80ee79c90c9250961c)